src/NodeManipulator/AssignManipulator.php

```diff
@@ -77,13 +77,9 @@ public function isLeftPartOfAssign(Node $node): bool
             /**
              * @var Assign|AssignOp|PreDec|PostDec|PreInc|PostInc $parentNode
              *
-             * Compare start token pos to ensure php_doc_info info not be checked
+             * Compare same node to ensure php_doc_info info not be checked
              */
-            if ($parentNode->var->getStartTokenPos() !== $node->getStartTokenPos()) {
-                return false;
-            }
-
-            return $this->nodeComparator->areNodesEqual($parentNode->var, $node);
+             return $this->nodeComparator->areSameNode($parentNode->var, $node);
```

Comment:
method should be named `isSameNode`

Reply:
The method is there for long time. I guess naming is fine, as "are" for "they are same" for compare both left and right are node.